### DISCUSSION TITLE
compatibility with Lumen 9.0

### DISCUSSION THREAD

**Discussion by evifere** [DirectoryTree/LdapRecord-Lumen#5] [Q&A]
Hi @stevebauman  have you plan to be compatible with lumen 9.0 soon ?

**Comment by stevebauman** [comment]
Sorry @evifere! I appreciate the reminder. I'll update it today with support and comment back here once done 👍

**Comment by stevebauman** [comment]
I've just released [v2.0.1](https://github.com/DirectoryTree/LdapRecord-Lumen/releases/tag/v2.0.1) with support for Lumen v9.0.

Thanks again for the report 🙏

**Reply by evifere** [reply]
thank you for making the change
